pkg/minikube/download/preload.go

```diff
@@ -91,6 +94,22 @@ func PreloadExists(k8sVersion, containerRuntime string) bool {
 		return false
 	}
 
+	cmd := exec.Command("docker", "info", "-f", "{{.Info.Driver}}")
```

Comment:
lets use the oci.DaemonInfo for this (add the storage driver to it)
(we can later make that a cached call so we dont have to keep calling it)

also lets try to download the image for that driver 
if that driver doesnt exist then dont preload
(we have the storage driver in the filename) so that shoudl make it easy.

so in the future if we mmake preload images for that storage driver we can suppor it
https://github.com/kubernetes/minikube/issues/7626#issuecomment-612580455

Reply:
here is the link for DaemonInfo

I believe u could update the struct with the new format, so the Storage Driver be unmarshalized into it

Reply:
I just copied the code from the generator

https://github.com/kubernetes/minikube/blob/master/hack/preload-images/preload_images.go#L82